test-clients/src/main/java/com/hedera/services/bdd/suites/file/FileUpdateSuite.java

```diff
@@ -772,6 +777,142 @@ private HapiApiSpec messageSubmissionSizeChange() {
+                        tokenCreate(ft)
+                                .treasury(GENESIS)
+                                .payingWith(GENESIS)
+                                .initialSupply(1_000_000)
+                                .exposingCreatedIdTo(id -> fungibleTokenId.set(asToken(id))),
+                        tokenCreate(nft)
+                                .treasury(GENESIS)
+                                .payingWith(GENESIS)
+                                .supplyKey(GENESIS)
+                                .tokenType(TokenType.NON_FUNGIBLE_UNIQUE)
+                                .initialSupply(0)
+                                .exposingCreatedIdTo(id -> nonFungibleTokenId.set(asToken(id))),
+                        mintToken(nft, List.of(ByteString.copyFromUtf8("HELLO"))))
+                .when(
+                        // Valid hollow creation via hbar transfer
+                        cryptoTransfer(
+                                        (spec, b) ->
+                                                b.getTransfersBuilder()
+                                                        .addAccountAmounts(
+                                                                aaWithNum(2, -ONE_HUNDRED_HBARS))
+                                                        .addAccountAmounts(
+                                                                aaWith(
+                                                                        aNewEvmAddress,
+                                                                        +ONE_HUNDRED_HBARS)))
+                                .signedBy(GENESIS)
+                                .via(uniqueCreation),
+                        getTxnRecord(uniqueCreation)
+                                .andAllChildRecords()
+                                .exposingChildCreationsTo(aCreatedId::set)
+                                .hasNonStakingChildRecordCount(1),
+                        getAccountInfoWithLiteralAlias(aNewEvmAddress)
+                                // QUESTION - should we expect .hasEmptyKey() here?
+                                .logged(),
+                        // Hollow account cannot be used as payer account
+                        withOpContext(
+                                (spec, opLog) ->
+                                        spec.registry()
+                                                .saveAccountId(STILL_HOLLOW, aCreatedId.get())),
+                        cryptoTransfer(tinyBarsFromTo(GENESIS, FUNDING, 1))
+                                .payingWith(STILL_HOLLOW)
+                                .fee(ONE_HBAR)
+                                .signedBy(GENESIS)
+                                .hasPrecheck(INVALID_SIGNATURE),
+                        cryptoTransfer(tinyBarsFromTo(STILL_HOLLOW, FUNDING, 1))
+                                .payingWith(GENESIS)
+                                .fee(ONE_HBAR)
+                                .signedBy(GENESIS),
```

Comment:
This one should also be failing with `INVALID_SIGNATURE`, because a hollow account cannot be a part of a transaction that needs its signature and not be the payer. So, even if this `cryptoTransfer` were signed by the `ECDSA` key related to a hollow account's `evmAddress`, it should still fail.

This scenario is returning `INVAID_SIGNATURE` as expected with the changes from the [finalization PR](https://github.com/hashgraph/hedera-services/pull/4128), where we return [a new `JHollowKey` for hollow accounts](https://github.com/hashgraph/hedera-services/pull/4128/files#diff-b098c669884e8d329f94aa569b67175407c425a13f17937fe7e8f3363f0dbfbbR224-R232) and it [cannot be verified](https://github.com/hashgraph/hedera-services/blob/1af08683a4a064b87e218bd625b5063f45391ba1/hedera-node/hedera-mono-service/src/main/java/com/hedera/services/keys/HederaKeyActivation.java#L126) when the `otherPartySigs` are being checked.

Reply:
Oh yes I apparently wanted to see how it failed and then didn't update the `hasKnownStatus()`!